src/main/java/dev/galacticraft/mod/content/GCBlockEntityTypes.java

```diff
@@ -31,103 +31,65 @@
+    public static final BlockEntityType<BasicSolarPanelBlockEntity> BASIC_SOLAR_PANEL = register(Constant.Block.BASIC_SOLAR_PANEL, BasicSolarPanelBlockEntity::new, GCBlocks.BASIC_SOLAR_PANEL);
+    public static final BlockEntityType<AdvancedSolarPanelBlockEntity> ADVANCED_SOLAR_PANEL = register(Constant.Block.ADVANCED_SOLAR_PANEL, AdvancedSolarPanelBlockEntity::new, GCBlocks.ADVANCED_SOLAR_PANEL);
 
     // WIRES, PIPES, WALKWAYS
-    public static final BlockEntityType<WireBlockEntity> WIRE_T1 = BlockEntityType.Builder.of((pos, state) -> WireBlockEntity.createT1(GCBlockEntityTypes.WIRE_T1, pos, state), GCBlocks.ALUMINUM_WIRE, GCBlocks.SEALABLE_ALUMINUM_WIRE).build();
-    public static final BlockEntityType<WireBlockEntity> WIRE_T2 = BlockEntityType.Builder.of((pos, state) -> WireBlockEntity.createT2(GCBlockEntityTypes.WIRE_T2, pos, state), /*GCBlocks.HEAVY_ALUMINUM_WIRE,*/ GCBlocks.HEAVY_SEALABLE_ALUMINUM_WIRE).build();
-    public static final BlockEntityType<GlassFluidPipeBlockEntity> GLASS_FLUID_PIPE = BlockEntityType.Builder.of(GlassFluidPipeBlockEntity::new, GCBlocks.GLASS_FLUID_PIPE).build();
-    public static final BlockEntityType<WalkwayBlockEntity> WALKWAY = BlockEntityType.Builder.of(WalkwayBlockEntity::new, GCBlocks.WALKWAY).build();
-    public static final BlockEntityType<WireWalkwayBlockEntity> WIRE_WALKWAY = BlockEntityType.Builder.of(WireWalkwayBlockEntity::new, GCBlocks.WIRE_WALKWAY).build();
-    public static final BlockEntityType<FluidPipeWalkwayBlockEntity> FLUID_PIPE_WALKWAY = BlockEntityType.Builder.of(FluidPipeWalkwayBlockEntity::new, GCBlocks.FLUID_PIPE_WALKWAY).build();
+    public static final BlockEntityType<WireBlockEntity> WIRE_T1 = register(Constant.Block.ALUMINUM_WIRE, (pos, state) -> WireBlockEntity.createT1(GCBlockEntityTypes.WIRE_T1, pos, state), GCBlocks.ALUMINUM_WIRE, GCBlocks.SEALABLE_ALUMINUM_WIRE);
+    public static final BlockEntityType<WireBlockEntity> WIRE_T2 = register(Constant.Block.HEAVY_ALUMINUM_WIRE, (pos, state) -> WireBlockEntity.createT2(GCBlockEntityTypes.WIRE_T2, pos, state), /*GCBlocks.HEAVY_ALUMINUM_WIRE,*/ GCBlocks.HEAVY_SEALABLE_ALUMINUM_WIRE);
+    public static final BlockEntityType<GlassFluidPipeBlockEntity> GLASS_FLUID_PIPE = register(Constant.Block.GLASS_FLUID_PIPE, GlassFluidPipeBlockEntity::new, GCBlocks.GLASS_FLUID_PIPE);
+    public static final BlockEntityType<WalkwayBlockEntity> WALKWAY = register(Constant.Block.WALKWAY, WalkwayBlockEntity::new, GCBlocks.WALKWAY);
+    public static final BlockEntityType<WireWalkwayBlockEntity> WIRE_WALKWAY = register(Constant.Block.WIRE_WALKWAY, WireWalkwayBlockEntity::new, GCBlocks.WIRE_WALKWAY);
+    public static final BlockEntityType<FluidPipeWalkwayBlockEntity> FLUID_PIPE_WALKWAY = register(Constant.Block.FLUID_PIPE_WALKWAY, FluidPipeWalkwayBlockEntity::new, GCBlocks.FLUID_PIPE_WALKWAY);
```

Comment:
These two lines are broken.

Reply:
Nevermind I'm just being stupid.

Reply:
Turns out it was an incompatibility between the pipes refactor and the block entity refactor. Huh.

Reply:
Time to update :)